easybuild/easyblocks/c/clang.py

```diff
@@ -198,7 +198,20 @@ def configure_step(self):
 
         # GCC and Clang are installed in different prefixes and Clang will not
         # find the GCC installation on its own.
-        self.cfg['configopts'] += "-DGCC_INSTALL_PREFIX='%s' " % get_software_root('GCC')
+        # First try with GCCcore, as GCC built on top of GCCcore is just a wrapper for GCCcore and binutils,
+        # instead of a full-fledge compiler
+        gcc_prefix = get_software_root('GCCcore')
+
+        # If that doesn't work, try with GCC
+        if gcc_prefix == None:
+            gcc_prefix = get_software_root('GCC')
+        
+        # If that doesn't work either, print error and exit
+        if gcc_prefix == None:
+            raise EasyBuildError("Can't find GCC or GCCcore to use")
+
+        self.cfg['configopts'] += "-DGCC_INSTALL_PREFIX='%s' " % gcc_prefix
+        self.log.debug("Using %s as GCC_INSTALL_PREFIX" % gcc_prefix)
```

Comment:
@damianam more style nitpicking: don't use `%` in `self.log.debug`, use `,`:

```python
self.log.debug("Using %s as GCC_INSTALL_PREFIX",  gcc_prefix)
```

(this results in only doing the actual templating when `self.log.debug` is called)